eng/common/scripts/job-matrix/README.md

```diff
@@ -101,7 +101,7 @@ Example:
   ],
   "framework": [
     "net461",
```

Comment:
Things under eng/common are only examples or tests so not necessary to fix and we cannot fix it in the .NET repo it would need to be updated in the tools repo.

Reply:
Yup, reverted.